### PR TITLE
Make changes_applied public again

### PR DESCRIPTION
**Pull request by shioyama**
This was (accidentally?) made private in #41860, whereas previously it was public. I'm making it public again.

### CODE REVIEW

**Comment by shioyama** [comment]
cc @p8

File: activerecord/lib/active_record/autosave_association.rb [hunk 1]
```diff
@@ -282,6 +282,11 @@ def changed_for_autosave?
       new_record? || has_changes_to_save? || marked_for_destruction? || nested_records_changed_for_autosave?
     end
 
+    def changes_applied
```
Comment (kamipo) [inline activerecord/lib/active_record/autosave_association.rb:285]:
```suggestion
    def changes_applied # :nodoc:
```

**Comment by p8** [comment]
@shioyama Ah, my mistake :)

**Comment by shioyama** [comment]
@p8 no worries, easy fix! :wink:

**Pull request merged**